### DISCUSSION TITLE
Should we move the `PyFixest` repository to a github organization?

### DISCUSSION THREAD

**Discussion by s3alfisc** [py-econometrics/pyfixest#444] [General]
Most successful open source project seem to be managed via a github organization. Would it make sense to move the pyfixest repository to a pyfixest org? What would be advantages and disadvantages?

Pro: 

- pyfixest is not tied to my private account - using an organization might signal long-term commitment of maintenance and might be more inclusive
- multiple modules in pyfixest could be standalone repositories (e.g. the `ccv` module, the `ritest` module, the `multcomp` module, the `did` module), just as [wildboottest ](https://github.com/s3alfisc/wildboottest). Moving these repositories out of pyfixest might allow for more general code that works with other regression packages (e.g. statsmodels). Additionally, it might decrease the CI runtime of the main repo, pyfixest, as most of the tests for e.g. the ccv repo would take place in the specialized repositories. This should speed up the development time of all new pyfixest features. 

Con: 
- Multiple distinct repositories will likely be harder to manage. 

@juanitorduz @NKeleher @aeturrell Do you have any thoughts / opinions / experiences on this topic? @amichuda, would you be open to move the `wildboottest` repo to a `pyfixest` org?

**Comment by juanitorduz** [comment]
Overall, I think it is a good idea! The PyFixest package is not that large now, and the syntax is consistent. Hence, in the short term, I do not think it is necessary to create many repositories, especially as it would require significant work. You could start by creating the GitHub organization and add these projects there as the effect of this does not seem that big :)

**Comment by amichuda** [comment]
Hi! This is a great project and idea. No issues with wildboottest being put under an org as long as I keep the same permissions 😀. 

But perhaps name it something more general? Theres a lot of interesting projects here that might not only be useful as modules in `pyfixest`. Maybe `fancyeconometrics` haha?

**Reply by s3alfisc** [reply]
Just checked and you should still have the same rights as before =)

**Comment by s3alfisc** [comment]
Thanks Aleks (@amichuda) - of course you will keep all your rights to the `wildboottest` repo. How about `py-econometrics` as a project name? I kind of like `fancy-econometrics`´but maybe it sounds too grandiose?

**Comment by s3alfisc** [comment]
I've now moved both `wildboottest` and `pyfixest` to the [py-econometrics](https://github.com/py-econometrics) organisation - happy to rename it in case we find a better name. @amichuda I have invited you as a member. After you accept, can you check if you have the same rights to wildboottest as before? @juanitorduz would you also be interested to be invited? Anyone else here? Also tagging @apoorvalal. If you respond with a thumbs-up, I'll send you an invitation =)

**Reply by apoorvalal** [reply]
Sounds great; thanks for the invite. I'd be happy to contribute my `GMM` and `fastmatch`  libraries you're open to having them. 

AL

**Reply by juanitorduz** [reply]
Thank you @s3alfisc for the invitation! I'll be happy to join even though I'am probably far away from a “core contributor” 😄.

**Reply by s3alfisc** [reply]
Fantastic, I've invited both of you! I really don't think there's a need to be a "core" contributor to be a part of the project @juanitorduz - you're contributions have been super valuable so far and I'd love to keep you engaged with pyfixest =) @apoorvalal, I'd be more than happy to include both `GMM` and `fastmatch`. I actually could have used a good matching estimator the other day at work but then went back to good old OLS 😄 I think you should be able to transfer them once you've joined the project?
